build.gradle

```diff
@@ -26,10 +27,15 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
 
 
-    compile group: 'com.google.gwt', name: 'gwt-user', version: '2.8.2'
-    compile group: 'com.google.gwt', name: 'gwt-dev', version: '2.8.2'
-    compile group: 'com.google.gwt', name: 'gwt-servlet', version: '2.8.2'
+    compile group: 'org.gwtproject.i18n', name: 'gwt-numberformat', version: '1.0-SNAPSHOT'
```

Comment:
make version a variable? for the ones that are all published together? i _think_ most of these are published together.

also, isn't at least one org.gwtproject.i18n artifact going to depend on this output, so this will end up being a little circular when we make releases?

Reply:
for the version thing, yes i will do.
for the second part no, the importer is currently an external tool, and by itself it does not publish any artifacts for other to depends on, it will just generate code into the gwt-cldr repo.